### ISSUE TITLE
#102 Top

### ISSUE THREAD

**Issue by SENOSuit** [HUNTERS1984/tenposs#10]
## Content

● Topページのレイアウトや構成を編集可能

**Comment by SENOSuit** [comment]
### 藤吉さん

topページのレイアウトというのは、複数テンプレがあり選択して変更可能という意味でしょうか？

構成は、ブロックを入れ替えるみたいなイメージでしょうか？

### 酒井

悩んでいます。
デザインやレイアウトは別のメニューとして設けるか。各機能ごとに設定させるか。

**Comment by ryo2mattock** [comment]
これは工数的にエンジニアに要確認ですが。有無で大きく工数変わるならばver2で良いです。初期はデザインを我々で変更するとかでも良いかと思います。

あと実装する際には、デザイン＞レイアウト　or　テンプレートで良いかと思います。

**Comment by SENOSuit** [comment]
レイアウト
= 優先度低

テンプレート
= 優先度高

デザイン（アイコンやフォント、カラーなど）
= 優先度高

という認識です。

**Comment by ryo2mattock** [comment]
そうですね、私も同意ですが。基本的に上記機能を実装するならばまとめてで良いと思います。なので今回のver1については、基本的にこちらで決めたレイアウト・デザインでからなども選択してもらい作るというアナログな手法で良いと思います。

**Comment by SENOSuit** [comment]
まとめてですと、事業者さんのブランディングの観点からあまりよろしくないように思います。

・レイアウト
これは、少々実装が複雑かつ工数がかかるので、ver2にて。

・テンプレート
これは、初期は3つほど作る形で。

・デザイン（アイコンやフォント、カラーなど）
これは、クラスを作るだけで実装も工数もかからないので、初期からいくつか選択可能にしといた方がいいと思います。

**Comment by ryo2mattock** [comment]
そうですね。確かに事業者からすると選択肢があると喜ばれると思います。ただこれはadminで変更ではなくて契約時のディレクションでヒアリングシートなどに記載してもらうアナログ形式でいいかなと思ってですね。確か、テンプレート選択して変更されるのも確か少々工数がかかったと思いますので。

**Comment by SENOSuit** [comment]
> 確か、テンプレート選択して変更されるのも確か少々工数がかかったと思いますので。

これ設計次第だと思うのですよね。
以前ってtenpossってVNにその辺の設計もさせていたんでしたっけ？

**Comment by ryo2mattock** [comment]
ある程度、こちらで指定してはいましたね。ただDBとかは向こうでやってましたね。

**Comment by SENOSuit** [comment]
成程、では、そのせいもあるかもしれません。
恐らく、各テンプレートを別ファイルで作れば、あとは読み込むファイルを指定するだけで済むと思われるので、工数かからないと思うのですよね。

**Comment by ryo2mattock** [comment]
なるほど、ではそこは工数をエンジニアさんと協議してきましょうか。軽いなら入れたほうが良いですしね。

**Comment by SENOSuit** [comment]
はい、これもエンジニアさんに要確認ですね。

**Comment by ryo2mattock** [comment]
はい。そうしましょう。
